### PR TITLE
VEN-1063 | Add an email template for order cancellation

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Add order cancellation email templates in all three languages 

### Closes :no_good_woman:
**[VEN-1063](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1063):** Order cancellation confirmation email

### CODE REVIEW

File: templates/email/messages/order_cancelled_en.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<p>Dear customer, we have received your cancellation. The invoice, application and agreement related to the berth are cancelled as of {{ order.rejectedAt }}.</p>
+
+<h3>Object</h3>
```
Comment (a-rmz) [inline templates/email/messages/order_cancelled_en.html:3]: `Object` sounds a bit odd in English. Not sure what would  be the best but it sounds a bit strange.

File: templates/email/messages/order_cancelled_en.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<p>Dear customer, we have received your cancellation. The invoice, application and agreement related to the berth are cancelled as of {{ order.rejectedAt }}.</p>
```
Comment (a-rmz) [inline templates/email/messages/order_cancelled_en.html:1]: The prop is on `snake_case`, same for the other templates. :)
```suggestion
<p>Dear customer, we have received your cancellation. The invoice, application and agreement related to the berth are cancelled as of {{ order.rejected_at }}.</p>
```

Comment (Nizar-Rahme) [reply templates/email/messages/order_cancelled_en.html]: Fixed

File: templates/email/messages/order_cancelled_en.html (hunk 1 above)
Comment (TatuArvela) [reply templates/email/messages/order_cancelled_en.html:3]: Should this be Paikka / Plats / Berth?

Comment (Nizar-Rahme) [reply templates/email/messages/order_cancelled_en.html:3]: It's `Kohde` in Finnish, there's `Paikka` a couple of lines below. What do you suggest @TatuArvela would be the right translation for `Kohde` in English? I also got translated `Område` in Swedish, do you think that's right?

Comment (TatuArvela) [reply templates/email/messages/order_cancelled_en.html:3]: I think we're using "Paikka / Plats / Place" as the generic term when talking about both berths and winter storage places, so we could consider using that.

If using "Kohde", I'd say "Location" would be a good English translation. "Område" means "area", but can make sense in this context I think.

Comment (Nizar-Rahme) [reply templates/email/messages/order_cancelled_en.html]: Fixed

**Review by a-rmz — APPROVED** [review]

**Pull request merged**